docs/csi_rn_supported_orchestration.md

```diff
@@ -0,0 +1,16 @@
+# Supported orchestration platforms
+
+The following table details orchestration platforms suitable for deployment of the IBM® block storage CSI driver.
+
+|Orchestration platform|Version|Architecture|
+|----------------------|-------|------------|
+|Kubernetes|1.19|x86|
+|Kubernetes|1.20|x86|
+|Red Hat® OpenShift®|4.6|x86, IBM Z®, IBM Power Systems™<sup>1</sup>|
+|Red Hat OpenShift|4.7|x86, IBM Z, IBM Power Systems<sup>1</sup>|
```

Comment:
any reason there are no marks (®,™) on the second line?

Reply:
aren't they required only for their first appearance in the page?

Reply:
as ori said. resolving this. thanks (it goes through a trademark scan)